bugbot/rules/crash_small_volume.py

```diff
@@ -180,6 +185,13 @@ def set_autofix(self, bugs):
                     )
                 )
                 autofix["keywords"] = {"remove": list(bug["keywords_to_remove"])}
+                autofix["flags"] = [
+                    {
+                        "id": flag_id,
+                        "status": "X",
+                    }
+                    for flag_id in bug.get("needinfos_to_remove", [])
```

Comment:
We do not need to use `.get()` since `needinfos_to_remove` will always be there:
```suggestion
                    for flag_id in bug["needinfos_to_remove"]
```

Reply:
Removed in f576b0743f2cefd117aaa169d3d41787bd824f7b!